static/styles/rendered_markdown.css

```diff
@@ -40,6 +40,15 @@
         margin-top: 0;
     }
 
+    /* Changing visibility of hr in light-mode */
+
+    hr {
+        margin: 20px 0;
+        border: 0;
```

Comment:
The above two properties are the default for `hr` in bootstrap, so you need not repeat it here.

Reply:
Ok let me amend.